environment.yml

```diff
@@ -2,36 +2,33 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - python=3.8
-  - cython==0.29.24
-  - pillow==8.3.2
+  - python=3.11
+  - cython==0.29.33
+  - pillow==9.4.0
   - pypubsub==4.0.3
-  - configparser==5.0.1
-  - h5py==2.10.0
-  - imageio==2.9.0
-  - nibabel==3.2.1
-  - numpy==1.21.2
-  - psutil==5.8.0
+  - h5py==3.8.0
+  - imageio==2.26.0
+  - nibabel==5.0.1
+  - numpy==1.24.3
+  - psutil==5.9.4
   - pyserial==3.5
-  - scikit-image==0.18.3
-  - scipy==1.7.1
-  - vtk==9.0.3
-  - wxpython==4.1.1
+  - scikit-image==0.20.0
+  - scipy==1.10.1
+  - vtk==9.2.6
+  - wxpython==4.2.0
```

Comment:
There is a newer version (4.2.1) and InVesalius is working with this version

Reply:
Thanks! I updated accordingly and specified the versions for the tracking devices packages.

Reply:
@tfmoraes can this be merged?

Reply:
👍